src/coord/src/coord.rs

```diff
@@ -250,6 +250,12 @@ struct PendingPeek {
     otel_ctx: OpenTelemetryContext,
 }
 
+struct PendingWriteTxn {
+    writes: Vec<WriteOp>,
+    sender: oneshot::Sender<Option<ExecuteResponse>>,
+    conn_id: u32,
+}
```

Comment:
Any docs you can put on here would be great. I'm guessing something about how `sender` should be told something only once the write is actually made durable?

Reply:
Just pushed some docs for  this, let me know if they are clear.